app/src/components/organisms/Header.tsx

```diff
@@ -41,6 +41,9 @@ function Header() {
           <Button onClick={navTo('/professorprofile')} color="inherit">
             Professor Profile
           </Button>
+          <Button onClick={navTo('/ErrorPage')} color="inherit">

```

Comment:
I would rather not leave a link to the error page in the navbar

Reply:
Good catch